library/src/main/java/com/github/pwittchen/reactivenetwork/library/network/observing/strategy/PreLollipopNetworkObservingStrategy.java

```diff
@@ -51,7 +51,12 @@
 
         subscriber.add(unsubscribeInUiThread(new Action0() {
           @Override public void call() {
-            context.unregisterReceiver(receiver);
+            //Catch case where the receiver is already unregistered
+            try {
+              context.unregisterReceiver(receiver);
+            } catch (Exception e) {
+              // ignore exception
```

Comment:
Thanks for the comments. I'll execute this code and merge that later. I'll add handling of this exception in the next commits because I have some sort of idea how to do it in a generic way, so library users can control that. Thanks for your contribution!